Bottom: 61f8c5dbe4fbd6855b3d99f1698a996df955cdd9
Top:    79ef10c7902ded4c1bc7f2e06b94221136c43019
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 17:13:51 +0000

Refresh of add-polygon-parentage-tree

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index e5efde6..7ff4d7c 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2632,6 +2632,8 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   if (ghidgui->debugged_polyarea != NULL) {
     PolygonType dummy_poly;
     dummy_poly.Clipped = ghidgui->debugged_polyarea;
+    dummy_poly.Flags = NoFlags ();
+    SET_FLAG (FULLPOLYFLAG, &dummy_poly);
 
     common_thindraw_pcb_polygon (Output.fgGC, &dummy_poly, &region);
     hidgl_flush_triangles (priv->hidgl);
